rita_client/src/dashboard/neighbors.rs

```diff
@@ -76,29 +76,18 @@ pub fn get_neighbor_info(
                     .from_err()
                     .and_then(move |stream| {
                         start_connection_legacy(stream)
-                            .from_err()
-                            .and_then(move |stream| {
-                                parse_routes_legacy(stream)
-                                    .from_err()
-                                    .and_then(|(_stream, routes)| {
-                                        let route_table_sample = routes;
-
-                                        NetworkMonitor::from_registry()
-                                            .send(GetStats {})
-                                            .from_err()
-                                            .and_then(|stats| {
-                                                let stats = stats.unwrap();
-                                                let output = generate_neighbors_list(
-                                                    stats,
-                                                    route_table_sample,
-                                                    combined_list,
-                                                );
-
-                                                Ok(Json(output))
-                                            })
-                                    })
-                                    .responder()
-                            })
+                        .from_err()
+                        .and_then(move |stream| {
+                            parse_routes_legacy(stream)
+                                .from_err()
+                                .and_then(|(_stream, routes)| {
+                                    let route_table_sample = routes;
+                                    let stats = get_stats(GetStats {});
```

Comment:
we can remove the GetStats{} argument here, it's no longer doing anything now that we don't need to attach an actix message trait implementation to it.